### ISSUE TITLE
Dataset vignette will not build on systems without Ghost Script installed

### ISSUE THREAD

**Issue by levisc8** [valentinitnelav/plotbiomes#2]
This particular vignette will not build on any system that doesn't already have Ghost Script installed. This is not a problem if we decide to keep the package on Github, but will be an issue if submitting to CRAN.

One alternative I've found is to use "static vignettes". In other words, create a pdf version of the vignette ahead of time and use that instead of having an RMarkdown file that is rebuilt each time it is installed/checked. They can still include code that is fully reproducible, but removes the Ghost Script dependency so that people who haven't installed yet can still download, build, and install the package. 

One drawback of this is that I'm not sure if we'd be able to include a dynamic vignette along with the static one, so we might be forced to change how the Examples vignette is produced as well. I don't think it would add too much work, but may be slightly annoying at first. 

`R.rsp` provides a mechanism for building static vignettes. See [this vignette](https://cran.r-project.org/web/packages/R.rsp/vignettes/R_packages-Static_PDF_and_HTML_vignettes.pdf) for details and let me know what you think.

**Comment by valentinitnelav** [comment]
For now I feel that is convenient to have the html documentation in the [html](https://github.com/valentinitnelav/plotbiomes/tree/master/html) folder of the repository. `.Rbuildignore` should make sure the files don't get caught in the building process. I am not keen on including PDF vignettes with the package, but having URLs pointing to html documentation is just fine with me and I saw this practiced by others as well (e.g. [mapview](https://cran.r-project.org/web/packages/mapview/index.html)). Otherwise, I might end up maintaining *.md files for PDF and HTML outputs which means extra work for basically the same thing - documenting, which I find it way more cool and fluid for reading in its html form than pdf form :) Besides, the main point of reproducibility for me is that I can click on something and remember what I did :)
